conf/test.conf

```diff
@@ -1,7 +1,7 @@
-# THIS IS NOT A PRODUCTION CONFIGURATION - DEVELOPMENT ONLY
+# THIS IS NOT A PRODUCTION CONFIGURATION - UNIT TEST ONLY
 # ~~~~~
 
-include "dev_base.conf"
+include "test_base.conf"
```

Comment:
oops, sorry! I figured i had enough of changing things for dev and accidentally breaking tests -_-